smallest_values/main.cpp

```diff
@@ -0,0 +1,81 @@
+#include <iostream>
+#include <cmath>
+#include <limits>
+
+double calculateMinimalValue(int const& size) {
```

Comment:
Here not required use const& type. Primitive types can't be used by const reference and can be passed by value. (std::string - not primitive type and can be huge, that's why we use const& or && for it where it can be available)